modin/core/storage_formats/pandas/query_compiler.py

```diff
@@ -1054,6 +1054,54 @@ def resample_var(self, resample_kwargs, ddof, *args, **kwargs):
     def resample_quantile(self, resample_kwargs, q, **kwargs):
         return self._resample_func(resample_kwargs, "quantile", q=q, **kwargs)
 
+    expanding_sum = Fold.register(
```

Comment:
FYI, ditto, there's no need to touch this file for pushdown work. We won't test changes to it anyway.

Reply:
Right, I added it as `expanding` and `rolling` are exactly the same from the Modin OSS algebra POV.